components/Item.js

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+
+export default function Item({ title, initialStatus }) {
+  return (
+    <StyledItem initialStatus={initialStatus}>
+      <Title>{title}</Title>
+    </StyledItem>
+  );
+}
+
+const StyledItem = styled.div`
+  width: 300px;
+  height: 70px;
```

Comment:
You can think about using relative units like em and rem as they have some advantages. However always make sure not to mix relative and absolute units with values that should scale in the same way.

Reply:
I did items size in em and responsive to text overflow now.